apm-agent-core/src/test/java/co/elastic/apm/agent/impl/transaction/FastExitSpanTest.java

```diff
@@ -70,8 +71,10 @@ void testExitSpanBelowDuration() {
         assertThat(spanCount.getDropped().get()).isEqualTo(1);
 
         DroppedSpanStats droppedSpanStats = reporter.getFirstTransaction().getDroppedSpanStats();
-        assertThat(droppedSpanStats.getStats("postgresql", Outcome.SUCCESS).getCount()).isEqualTo(1);
-        assertThat(droppedSpanStats.getStats("postgresql", Outcome.SUCCESS).getSum()).isEqualTo(49_999L);
+
+        ServiceTarget st = new ServiceTarget().withType("postgresql");
+        assertThat(droppedSpanStats.getStats(st, Outcome.SUCCESS).getCount()).isEqualTo(1);
+        assertThat(droppedSpanStats.getStats(st, Outcome.SUCCESS).getSum()).isEqualTo(49_999L);
```

Comment:
Those are a lot of changes instead of doing `ServiceTarget st = new ServiceTarget()` within `DroppedSpanStats#getStats` and leave all tests the same